.travis.yml

```diff
@@ -5,7 +5,7 @@ sudo: false
 env:
   global:
     - INFECTION_FLAGS='--threads=4 --min-msi=67 --min-covered-msi=75 --coverage=coverage --log-verbosity=none'
-    - PHPUNIT_BIN='vendor/bin/phpunit --prepend=devTools/xdebug-filter.php --coverage-clover=clover.xml --coverage-xml=coverage/coverage-xml --log-junit=coverage/phpunit.junit.xml'
+    - PHPUNIT_BIN='vendor/bin/phpunit --prepend=devTools/xdebug-filter.php --coverage-clover=clover.xml --coverage-xml=coverage/coverage-xml --log-junit=coverage/junit.xml'
```

Comment:
Could this change be done in a separate PR?

Reply:
well, I extracted a couple of PRs from here. Others: benefits at this point of time, when it is already here and coupled to the changes in this PR